src/component/base/DraftEditorProps.js

```diff
@@ -98,6 +98,11 @@ export type DraftEditorProps = {
 
   handlePastedFiles?: (files: Array<Blob>) => boolean,
 
+  // Handle dropped files
+  handleDroppedFiles?: (selection: SelectionState, files: Array<Blob>) => boolean,
+  // Handle other drops to prevent default text movement/insertion behaviour
+  handleDrop?: (selection: SelectionState, dataTransfer: DataTransfer, isInternal: boolean) => boolean,
```

Comment:
It might seem an odd request, but I really dislike boolean arguments. Can you create a module that defines a Flow type to denote this value?

```javascript
@providesModule DraftDragType

export type DraftDragType = 'internal' | 'external';
```

Then,

```javascript
const dragType = this._internalDrag ? 'internal' : 'external';
if (this.props.handleDrop && this.props.handleDrop(dropSelection, data, dragType) {
  ...
}
```

I do something similar for `DraftRemovalDirection`.

In truth I don't like boolean return values on these handlers either, so I should really clean up at some point too. :)